### PR TITLE
Bump pullrequest-init and kubeconfigwriter version to v0.20.1

### PR DESCRIPTION
**Pull request by barthy1**
# Changes

Latest image versions have multi-arch support, it will allow to use the same catalog tasks for many hardware architectures.

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality of task changed or new task added)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)
- [ ] Complies with [Catalog Organization TEP][TEP], see [example]. **Note** [An issue has been filed to automate this validation][validation]
  - [ ] File path follows  `<kind>/<name>/<version>/name.yaml`
  - [ ] Has `README.md` at `<kind>/<name>/<version>/README.md`
  - [ ] Has mandatory `metadata.labels` - `app.kubernetes.io/version` the same as the `<version>` of the resource
  - [ ] Has mandatory `metadata.annotations` `tekton.dev/pipelines.minVersion`
  - [ ] mandatory `spec.description` follows the convention

          ```

          spec:
            description: >-
              one line summary of the resource

              Paragraph(s) to describe the resource.
          ```

_See [the contribution guide](https://github.com/tektoncd/catalog/blob/master/CONTRIBUTING.md)
for more details._

---

[TEP]: https://github.com/tektoncd/community/blob/master/teps/0003-tekton-catalog-organization.md
[example]: https://github.com/tektoncd/catalog/tree/master/task/git-clone/0.1
[validation]:  https://github.com/tektoncd/catalog/issues/413

### CODE REVIEW

**Comment by chmouel** [comment]
/lgtm

thanks

**Review by Divyansh42 — APPROVED** [review]
/lgtm

File: task/kubeconfig-creator/0.1/kubeconfig-creator.yaml [hunk 1]
```diff
@@ -61,7 +61,7 @@ spec:
     - name: output
   steps:
     - name: write
-      image: gcr.io/tekton-releases/github.com/tektoncd/pipeline/cmd/kubeconfigwriter:v0.16.2
+      image: gcr.io/tekton-releases/github.com/tektoncd/pipeline/cmd/kubeconfigwriter:v0.20.1
```
Comment (vinamra28) [inline task/kubeconfig-creator/0.1/kubeconfig-creator.yaml:64]: can we tag the image with digest as well?

File: task/pull-request/0.1/pull-request.yaml [hunk 2]
```diff
@@ -35,7 +35,7 @@ spec:
   - name: pr
   steps:
   - name: pullrequest-init
-    image: gcr.io/tekton-releases/github.com/tektoncd/pipeline/cmd/pullrequest-init:v0.16.2
+    image: gcr.io/tekton-releases/github.com/tektoncd/pipeline/cmd/pullrequest-init:v0.20.1
```
Comment (vinamra28) [inline task/pull-request/0.1/pull-request.yaml:38]: same as above

**Review by vinamra28 — CHANGES_REQUESTED** [review]

File: task/kubeconfig-creator/0.1/kubeconfig-creator.yaml (hunk 1 above)
Comment (barthy1) [reply task/kubeconfig-creator/0.1/kubeconfig-creator.yaml]: done

**Comment by vinamra28** [comment]
/lgtm
Thanks :+1:

**Pull request merged**